src/xdocs/config_naming.xml

```diff
@@ -1179,6 +1179,18 @@ public boolean equals(Object o) {
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td><code>false</code></td>
           </tr>
+          <tr>
+            <td>scope</td>
+            <td>Visibility scope of methods where parameters are checked.</td>
+            <td><a href="property_types.html#scope">scope</a></td>
+            <td><code>anoninner</code></td>
+          </tr>
+          <tr>
+            <td>excludeScope</td>
+            <td>Visibility scope of methods where parameters are not checked.</td>
+            <td><a href="property_types.html#scope">scope</a></td>
+            <td><code>null</code></td>
+          </tr>
```

Comment:
@agcuda The examples you added went in the description where as we have a sub-section specifically for example configurations. Please move your examples to that sub-section and give them a brief 1 liner to describe what the example accomplishes.